projects/canopy/src/lib/carousel/auto-play/auto-play.component.scss

```diff
@@ -0,0 +1,16 @@
+.lg-carousel-autoplay {
+  position: absolute;
+  right: var(--space-sm);
+  z-index: var(--z-index-autoPlay);
```

Comment:
I can't see this variable anywhere? the main.scss has `z-index` which I'm assuming has to be matching this?

Could you change the name to be `--z-index-carousel-autoplay` please?